spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -379,6 +379,53 @@ definitions:
             fill: STECSatElement
             desc: Array of STEC polynomial coefficients for each space vehicle.
 
+ - BoundsHeader:
+     short_desc: Header for the Bounds messages
+     fields:
+       - time:
+           type: GPSTimeSec
+           desc: GNSS reference time of the bound
+       - num_msgs:
+           type: u8
+           desc: Number of messages in the dataset
+       - seq_num:
+           type: u8
+           desc: Position of this message in the dataset
+       - update_interval:
+           type: u8
+           desc: >
+             Update interval between consecutive bounds. Similar to
+             RTCM DF391.
+       - sol_id:
+           type: u8
+           desc: SSR Solution ID.
+
+ - MSG_SSR_STEC_CORRECTION_NEW:
```

Comment:
I'll look for a better name to differentiate it with the existing message, _NEW isn't the best of suffixes

Reply:
Standard procedure here is to add `_DEP` (or `_DEP_A` if `_DEP` already exists) for the existing message and have this new message occupy the old name. I would go ahead and do that in this PR since these messages should be treated like the real thing.

Reply:
And what happens with the IDs of the messages?
The message validator threw the following `Previous ID: 1531, Current ID: 1533`, so I guess I should have kept the original ID.

Reply:
@isaactorz ^^ this is a bug with the message validator that we didn't notice during review, it's a common pattern to rename messages (the ID and contents of the message must remain the same)